Added GDPR modal

njrenaissance opened the pull request:
* Added Cookies Policy Page
* Fixed linting issue on my end with <CR><LF> in Windows

kirlen commented:
@ameer-clara and @codiak jonathan and i walked through this saturday. code looks fine and all, but reviewing the wording, we found ourselves wondering: does it really make sense to introduce this modal right now, before we are actually doing anything conditionally based on user consent? was there an assumption that this would be tied to our use of google analytics (which i'm not sure would be necessary, as long as no identifying info is being used)? if we don't have a concrete use for it, not sure it's a plus to just toss it in there without anything specific to tie the wording to. thoughts?

ameer-clara commented:
Thanks for following up on this Kevin and Jon. my 2c, you can play it any
way you like.

I think this is super important especially since we are funded by a
European org and are in a space where privacy matters.

Check bbc for inspiration, turn on your vpn (point to uk) and visit bbc,
see their options..

my rough thoughts around this:

I love tracking and it helps so much to show a viable product, from
investors to product dev. Whenever you request for funding, the investors
want to see your tracking metrics, that is their goto step to measure
traction.

default setting - we track you, where you visiting from, browser, os,
userid of twitter archive, size of twitter archive, how long it takes to
upload (successfully vs failed)  user flow, workflow from initial visit to
upload to archive in swarm, time spent on pages etc, DAU
vs private - no country, browser, os tracking, size of archive and user
flow
vs super private - we don't track you at all, no analytics

Would love to get your point of view, Cody and both of your opinion on this
and next steps.

Thanks!



On Mon, Apr 18, 2022 at 5:26 AM Kevin Irlen ***@***.***>
wrote:

> @ameer-clara <https://github.com/ameer-clara> and @codiak
> <https://github.com/codiak> jonathan and i walked through this saturday.
> code looks fine and all, but reviewing the wording, we found ourselves
> wondering: does it really make sense to introduce this modal right now,
> before we are actually doing anything conditionally based on user consent?
> was there an assumption that this would be tied to our use of google
> analytics (which i'm not sure would be necessary, as long as no identifying
> info is being used)? if we don't have a concrete use for it, not sure it's
> a plus to just toss it in there without anything specific to tie the
> wording to. thoughts?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/codiak/socialarchive-dapp/pull/82#issuecomment-1101328151>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOQPKPZDX7IE465I6ORIKGLVFVBFTANCNFSM5TRLF5IQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

njrenaissance commented:
I updated my pull request.  One change was removing the unused context-related functions in cookie-consent.tsx.  However, the other two changes were wording for the Cookies popup and the Policy page.  I tried to be as honest as possible about what we are collecting.

I removed any language that suggested any downside to not accepting cookies.  Realistically there isn't.  It's bad for us maybe but not for the user.  I didn't go into detail on each and every cookie because I don't know what that would be yet, but some sites explain every cookie.
